DEVELOPMENT-GUIDE.md

```diff
@@ -1,8 +1,9 @@
 # ACCP Development Guidance
-Correctness and code-safety is paramount in the Amazon Corretto Crypto Provider.
+Correctness and code safety is paramount in the Amazon Corretto Crypto Provider.
 
-The purpose of this guide is not to provide all information needed for development within ACCP.
-Instead, it is intended to provide a quick introduction to the most important (and ACCP-specific) components to help developers find and learn what is most important.
+This guide is designed to provide a quick introduction to the most important (and ACCP-specific) components to help developers find and learn what is most important.
+The purpose of this guide is not to provide all information needed to develop ACCP.
```

Comment:
This can be interpreted as "The purpose of this guide is to withhold information." 

It doesn't provide ...